src/main/resources/schema.sql

```diff
@@ -0,0 +1,60 @@
+CREATE TABLE users (
+    id UUID PRIMARY KEY,
+    created_at TIMESTAMPTZ NOT NULL,
+    updated_at TIMESTAMPTZ,
+    username VARCHAR(50) UNIQUE NOT NULL,
+    email VARCHAR(100) UNIQUE NOT NULL,
+    password VARCHAR(60) NOT NULL,
+    profile_id UUID REFERENCES binary_contents(id) ON DELETE SET NULL
+);
+
+CREATE TABLE binary_contents (
+    id UUID PRIMARY KEY,
+    created_at TIMESTAMPTZ NOT NULL,
+    file_name VARCHAR(255) NOT NULL,
+    size BIGINT NOT NULL,
+    content_type VARCHAR(100) NOT NULL,
```

Comment:
마지막에 콤마를 제거해줘야 sql문이 정상동작할 것 같아요.
```suggestion
    content_type VARCHAR(100) NOT NULL
```